### DISCUSSION TITLE
Trolley Overlap

### DISCUSSION THREAD

**Discussion by LuigiMdg** [PrestaShop/PrestaShop#30864] [Q&A]
### Describe the bug and add attachments

Since I updated to _1.7.8.8_, I keep getting these errors.
![image](https://user-images.githubusercontent.com/18232125/212309307-1749c682-2d5c-4934-99e9-756d972cdd22.png)
I guess customers are having trouble buying.....

### Expected behavior

That does not give errors

### Steps to reproduce

Install _PrestaShop 1.7.8.5_, then update to the latest version.

### PrestaShop version(s) where the bug happened

1.7.8.8

### PHP version(s) where the bug happened

7.4

**Comment by kpodemski** [comment]
Hi @LuigiMdg 

The steps you provided:
> Install PrestaShop 1.7.8.5, then update to the latest version.

Are for 100% not going to cause the same error as you have. I compared those two versions:
https://github.com/PrestaShop/PrestaShop/compare/1.7.8.5...1.7.8.8

And there are no critical changes in the checkout mechanism. I'm sure that along with the core upgrade, you upgraded some of the payment modules, and maybe there's some issue with one of them.

Finally:

> I guess customers are having trouble buying.....

You can check this by creating test orders yourself :)

**Comment by LuigiMdg** [comment]
But no.. Sorry to disappoint you, but I only update the modules every few months, just to minimize the resulting problems.

**Reply by kpodemski** [reply]
Did you try to place a test order? :) At the moment there's absolutely no reason to say that "it's a Core fault"

**Reply by LuigiMdg** [reply]
I never said it was a fundamental error, but I think you wouldn't like receiving error emails every few hours either..
Anyway, no, I haven't tried it, but it's happened before with multiple customers/carts.
